test/functional_cpu_test.py

```diff
@@ -300,19 +300,23 @@ def test_linearity_of_istft4(self):
 
 class TestDetectPitchFrequency(common_utils.TorchaudioTestCase):
     def test_pitch(self):
```

Comment:
Since this test is now parameterized by frequency (100 and 440) can you use `parameterized.expand`? that will simplify the test implementation.